dev-python/libsass/libsass-0.19.4.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="A straightforward binding of libsass for Python"
+HOMEPAGE="https://github.com/sass/libsass-python"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
```

Comment:
It looks like Github has some test cases available, could you investigate possibility in enabling them?

Reply:
I did look into this earlier, but the tests require `sassc`, which is the file I patched out.

Also, the github tarball is incomplete (and therefore doesn't compile), it's missing the `.libsass-upstream-version` file.

Reply:
That should be an easy sed one-liner ;) if the pypi package has tests, do enable them! Helps the maintenance in the long run a lot.

Reply:
Or did I get the patch wrong actually, I thought it just renames the binary, not disables it. Didn't pysassc work equally to sassc?

Reply:
> Or did I get the patch wrong actually, I thought it just renames the binary, not disables it. Didn't pysassc work equally to sassc?

Upstream is deprecating `sassc` in steps, currently (without patch) it installs `pysassc` and `sassc` I have patched out `sassc.py` and patched `setup.py` to not look for it.

I'm currently trying to patch `sasstest.py` to see if I can get it to work without `sassc`, in theory it should because `pysassc` is basically the same thing.

Reply:
> It looks like Github has some test cases available, could you investigate possibility in enabling them?

I think I got it working, it runs `sasstest.py` and it completes without errors so i guess it's working.

Reply:
> It looks like Github has some test cases available, could you investigate possibility in enabling them?

I think I got it working, it runs `sasstest.py` and completes without errors, so I guess it works now.